hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/security/TestAWSV4AuthValidator.java

```diff
@@ -64,15 +67,74 @@ public static Collection<Object[]> data() {
                 "577efef23edd43b7e1a59",
             "5d672d79c15b13162d9279b0855cfba" +
                 "6789a8edb4c82c400e06b5924a6f2b5d7",
-            "wJalrXUtnFEMI/K7MDENG+bPxRfiCYEXAMPLEKEY"
+            "wJalrXUtnFEMI/K7MDENG+bPxRfiCYEXAMPLEKEY",
+            true
+        },
+        // Invalid Algorithm
```

Comment:
New test case: invalid signature

```suggestion
        },
        // Invalid signature
        {
            "AWS4-HMAC-SHA256\n" +
                "20150830T123600Z\n" +
                "20150830/us-east-1/iam/aws4_request\n" +
                "f536975d06c0309214f805bb90ccff089219ecd68b2" +
                "577efef23edd43b7e1a59",
            "5d672d79c15b13162d9279b0855cfba" +
                "6789a8edb4c82c400e06b5924a6f2b5d8",
            "wJalrXUtnFEMI/K7MDENG+bPxRfiCYEXAMPLEKEY",
            false
        },
        // Invalid Algorithm
```